### PR TITLE
fieldset pattern for radios, checkboxes

### PR DESCRIPTION
**Pull request by ajb**
Alternative to #243.

As you can see, the only change in our markup is to add `wrapper: :vertical_fieldset` or `wrapper: :horizontal_fieldset` to the input. Here's a screenshot of the generated markup: http://take.ms/f40Z5.

@jrubenoff: Let me know what you think?

I didn't include the date/time picker in the PR, since it's a separate concern. I'll get to it shortly, though.

### CODE REVIEW

**Comment by ajb** [comment]
Just kidding -- I added the pattern to the datepicker, too.

**Comment by jrubenoff** [comment]
Looks good!

**Comment by ajb** [comment]
Great. I will refactor and get this in.

File: spec/dummy/app/assets/javascripts/preview.coffee [hunk 1]
```diff
@@ -8,9 +8,6 @@ $ ->
 $(document).on 'click', '.docs_toggle_button', ->
   $(@).closest('.docs_item').find('.docs_code').toggle()
 
-$(document).on 'update.dtpicker', '#datetime_picker_pick_a_date_and_time', (_, ts) ->
```
Comment (ajb) [inline spec/dummy/app/assets/javascripts/preview.coffee]: Next time, ask before removing? I added this specifically in a PR: https://github.com/dobtco/dvl-core/pull/181. The goal was to show developers what value would be passed to the server.

**Pull request merged**